README.md

```diff
@@ -15,11 +15,30 @@ Please ensure all tasks have a description that includes a clear definition of w
 
 Builds are run automatically out of this repository. Basic workflow is:
 
-- Create a branch and implement your changes
-- The Docker image is built on the branch and as part of the pull request
-- Once the PR is merged, a build will be run and pushed to `latest` and `sudo-latest` from master
-- When a new stable image is ready, create a release with a version number like `0.0.6` and
-this will be pushed to `0.0.6` and `sudo-0.0.6`.
+- Create a branch and implement your changes.
+- The Docker image is built on the branch and as part of the pull request.
+- The integration test will be run on PR, merge and release events.
+- Once the PR is merged, a build will run. The resulted image will be tagged with git `head` and `latest` and pushed to ECR.
+- When decide to release a new stable image, create a release with a version number with `major.minor.patch`, e.g.,  `2.0.1`.
+It will trigger the build and image will be tagged with `2.0.1` and `stable` and pushed to ECR.
+
+## Packages' version maintenance and upgrade
+
+The base environment is using conda and docker image is built in two stages:
```

Comment:
```suggestion
The base environment uses Conda, and builds the Docker image in two stages:
```

Reply:
If the base environment is not the one that builds the Docker image, then better to say 'and the docker image is built..."

Reply:
Hmm. Just realising I'm also confused. Base environment inside the first stage docker build... ?

Reply:
While I assume you all know how to build a docker image, and I seriously doubt if more words will help, I have no better way but sound like giving a  lesson on "how to build a docker image", and I apologise.

- the docker image has to build upon on another image, at least in our case. This "another image" serves as the base environment for further customised installation, which could be ubuntu or conda or gdal or anything else. For this sandbox image, conda is used. So the base environment is only the component and it does NOT DO anything itself. 
- multi-stage builds, ref https://docs.docker.com/build/building/multi-stage/

Reply:
@emmaai sorry I got too focussed on the phrasing, and not focussed enough on making sure I actually understood the [process](https://github.com/GeoscienceAustralia/dea-sandbox/blob/223276e1c5dd6665ff52a6180b98ea3caba68c50/docker/Dockerfile#LL27C37-L27C44). Thanks for that Docker doco as a refresher.